osu.Game/Overlays/Settings/Sections/Input/KeyBindingsSubsection.cs

```diff
@@ -34,10 +34,9 @@ private void load(RealmContextFactory realmFactory)
         {
             string rulesetName = Ruleset?.ShortName;
 
-            List<RealmKeyBinding> bindings;
+            List<RealmKeyBinding> bindings = null;
 
-            using (var realm = realmFactory.CreateContext())
-                bindings = realm.All<RealmKeyBinding>().Where(b => b.RulesetName == rulesetName && b.Variant == variant).Detach();
+            realmFactory.Run(realm => bindings = realm.All<RealmKeyBinding>().Where(b => b.RulesetName == rulesetName && b.Variant == variant).Detach());
```

Comment:
reads better having `bindings` outside the method:
```suggestion
            bindings = realmFactory.Run(realm => realm.All<RealmKeyBinding>().Where(b => b.RulesetName == rulesetName && b.Variant == variant).Detach());
```